modules/swagger-codegen/src/main/resources/swift3/AlamofireImplementations.mustache

```diff
@@ -161,6 +161,13 @@ open class AlamofireRequestBuilder<T>: RequestBuilder<T> {
                     return
                 }
 
+                // handle HTTP 204 No Content
+                // NSNull would crash decoders
+                if response.response?.statusCode == 204 && response.result.value is NSNull{
+                    completion(nil, nil)
+                    return;
+                }
```

Comment:
Yes, I only had to prevent returning `NSNull` and converting it to `nil`. NSNull is only a placeholder for nil used by the JSON parser, as arrays and dictionaries in objc were not able to contain nil values.